challenges/objects-arrays.js

```diff
@@ -8,27 +8,46 @@
 
 // tyrannosaurus, carnivorous, 7000kg, 12m, Late Cretaceous
 
+let tyrannosaurus = {
+  name: 'tyrannosaurus',
+  diet: 'carnivorous',
+  weigh: '7000kg',
+  length: '12m',
+  period: 'Late Cretaceous',
+  'roar': function () {
```

Comment:
Quotes on *roar* are optional. But in general it is good practice to make all the keys the same within each object